Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -1118,6 +1118,63 @@ There are no input arguments for this command.
 
 There is no context output for this command.
 
+### microsoft-teams-token-permissions-list
+***
+Retrieves the API permissions associated with the used graph access token. 
+
+Use this command if you encounter insufficient permissions error when attempting to execute an integration command. Compare the permissions list obtained for the token with the permissions required for the desired command (can be found in the integration documentation), if there are missing API permissions, add them to your application.
```

Comment:
In case of missing permissions, they will need to reset auth - maybe we should mention this also here https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams#microsoft-teams-auth-reset

Reply:
Thanks! added both here and in the YML file.